src/segment-loader.js

```diff
@@ -938,6 +938,8 @@ export default class SegmentLoader extends videojs.EventTarget {
     if (segmentInfo.timestampOffset !== null &&
         segmentInfo.timestampOffset !== this.sourceUpdater_.timestampOffset()) {
       this.sourceUpdater_.timestampOffset(segmentInfo.timestampOffset);
+      // fired when a timestamp offset is set in HLS (can also identify discontinuities)
+      this.trigger('hlstimestampoffset');
```

Comment:
since we control this event and its being triggered on the `SegmentLoader` and not the `tech`, we don't need to have the `hls` prefix. But the `usage` event fired on the `tech` should still have the `hls-` prefix